src/devices/Arduino/README.md

```diff
@@ -0,0 +1,106 @@
+            using (var port = new SerialPort(portName, 115200))
+            {
+                Console.WriteLine($"Connecting to Arduino on {portName}");
+                try
+                {
+                    port.Open();
+                }
+                catch (UnauthorizedAccessException x)
+                {
+                    Console.WriteLine($"Could not open COM port: {x.Message} Possible reason: Arduino IDE connected or serial console open");
+                    return;
+                }
+
+                ArduinoBoard board = new ArduinoBoard(port.BaseStream);
+                try
+                {
+                    board.LogMessages += BoardOnLogMessages; // Get log messages
+                    board.Initialize();
+                    Console.WriteLine($"Connection successful. Firmware version: {board.FirmwareVersion}, Builder: {board.FirmwareName}");
+                    // Add code that uses the board here.
+                }
+                catch (TimeoutException x)
+                {
+                    Console.WriteLine($"No answer from board: {x.Message} ");
+                }
+                finally
+                {
+                    port.Close();
+                    board?.Dispose();
+                }
+            }
+ ```
+ 
+On Windows, only one application can use the serial port at a time, therefore you'll get "permission denied" errors when you try to run your program while i.e. the Serial Port Monitor of the Arduino IDE is open when you start your program. On the other hand, trying to upload a new sketch while the program runs will also fail. Note that the serial port monitor is of little use when the firmata firmware is loaded, since the communication protocol uses a binary format. Most of the output therefore will look like garbage. 
+
+## Known limitations
+
+All communication is routed trough the USB cable immitating a serial port with a limited bandwith. Therefore, some not insignificant delays are to be expected when sending commands or retrieving data. Communicating with sensors which have time-critical behavior will most likely not work reliably for this reason and the standard bindings provided for these won't work. This includes sensors like the DHT11, DHT22 or HCSR-04. For some of these, special Firmata modules are available to execute the time-critical part directly on the Arduino. This problem does not exist for sensors using I2C or SPI protocols. 
+
+For the moment this binding supports GPIO, Analog In, SPI, I2C and DHT on all platforms. Here is the list of tested features:
+
+- [x] SPI master support for Windows 64/32
+- [x] I2C master support for Windows 64/32
+- [x] Basic GPIO support for Windows 64/32
+- [x] Advanced GPIO support for Windows 64/32
+- [x] Analog input support on Windows 64/32
+- [ ] SPI support for MacOS 
+- [ ] I2C support for MacOS
+- [ ] GPIO support for MacOS
+- [ ] SPI support for Linux 64
+- [ ] I2C support for Linux 64
+- [ ] GPIO support for Linux 64
```

Comment:
I'd recommend removing this. You haven't used any interop so this should technically just work. If not for some mysterious reason then someone will create an issue